Cargo.toml

```diff
@@ -22,10 +22,11 @@ decimal     = { version = "2.0", default-features = false, optional = true }
 num-complex = { version = "0.2", default-features = false }
 packed_simd = { version = "0.3", optional = true }
 wide        = { version = "0.4", optional = true }
-fixed       = { version = "1", optional = true }
+fixed       = { version = "1", features = [ "serde" ], optional = true }
```

Comment:
The `serde` feature should not always be enabled. We should have a `serde-serialize` features like this:

```
serde-serialize = [ "serde", "fixed/serde" ]
```